src/test/kotlin/no/nav/syfo/syfosmvarsel/nysykmelding/NySykmeldingServiceKtTest.kt

```diff
@@ -79,10 +77,10 @@ object NySykmeldingServiceKtTest : Spek({
             oppgavevarsel.ressursId shouldEqual sykmelding.sykmelding.id
             oppgavevarsel.mottaker shouldEqual "123124"
             oppgavevarsel.utlopstidspunkt shouldBeAfter oppgavevarsel.utsendelsestidspunkt
-            oppgavevarsel.varseltypeId shouldEqual "NySykmelding"
+            oppgavevarsel.varseltypeId shouldEqual "NySykmeldingUtenLenke"
             oppgavevarsel.varselbestillingId shouldNotBe null
-            oppgavevarsel.utsendelsestidspunkt shouldBeAfter LocalDate.now().atTime(8, 59)
-            oppgavevarsel.utsendelsestidspunkt shouldBeBefore LocalDate.now().plusDays(1).atTime(17, 0)
+            // oppgavevarsel.utsendelsestidspunkt shouldBeAfter LocalDate.now().atTime(8, 59)
```

Comment:
Kanskje fjerne dei, isteden for å kommenter ut?

Reply:
Det var bare for å teste, og den er revertert nå?